### PR TITLE
Implement Create Contract Max codesize Tests

### PR DESCRIPTION
**Pull request by marioevz**
This PR adds two test cases where the max codesize contract is created, implementing #985.
The two new test files implement CREATE and CREATE2, both have 4 test cases each, where two are positive (codesize <= max codesize, contract is created), and two negative (codesize > max codesize, contract is not created).

PR is not ready since the last test case mentioned in #985 is not yet implemented.

### CODE REVIEW

**Comment by marioevz** [comment]
Added max codesize contract with selfdestruct test case.

File: src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      code: ''
+      nonce: '0'
+      storage: {}
+
+    00000000000000000000000000000000deadbeef:
+      balance: 2
+      code: |
+        :yul
+        {
+          // initcode: { push2(0x6000), dup1, push1(0x0c), push1(0x00), codecopy, push1(0x00), return,
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml:24]: need an example where it does not suicide. to see that this code does actually creates a contract with max allowed codesize

**Comment by winsvega** [comment]
also tests are not regenerated (new tests)

File: src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml (hunk 1 above)
Comment (marioevz) [reply src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml:24]: I think that should be covered by CreateMaxCodesize and Create2MaxCodesize, they both have a test case where the codesize is exactly 0x6000, and another pair of test cases where the codesize is exactly 0x6001.

Comment (winsvega) [reply src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeSuicideFiller.yml:24]: In Andrew's tests it was very clear. here is a bit hard to read

File: src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeFiller.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    nonce: '0'
+    to: 00000000000000000000000000000000deadbeef
+    secretKey: "45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8"
+    value:
+    - 0
+
+  expect:
+    - indexes:
+        data:
+        - !!int 0
```
Comment (winsvega) [inline src/GeneralStateTestsFiller/stCreateTest/CreateMaxCodesizeFiller.yml:57]: you can use labels

**Comment by winsvega** [comment]
I think Adrew pretty much covered this cases.

**Comment by marioevz** [comment]
Agree that #986 covers this already. Closing.

**Pull request closed without merging**